src/audio/mod.rs

```diff
@@ -0,0 +1,9 @@
+//! Loading and playing of audio files.
+pub mod play;
+pub mod output;
+
+mod audio_context;
+mod source;
+
+pub use self::audio_context::AudioContext;
```

Comment:
I like to keep my mod files as clean as possible, make them only describe the hierarchy and exports.  Makes the code a little easier to read in my opinion.  What do you think of that?